docs/how_tos/adding_custom_fields_to_the_registration.rst

```diff
@@ -0,0 +1,116 @@
+======================================================
+Adding Custom Fields to the Registration Page in Authn
+======================================================
+
+Introduction
+------------
+This guide explains how to add custom fields to the registration page of your Open edX instance. To integrate custom fields with the new Authn MFE, additional configuration steps are required. Custom fields can be either required—added directly to the registration page—or optional, and will be added to a secondary registration page that users can skip.
+
+Two main ways to add additional fields depending on the type of fields you want
+------------
+Open edX platform has default additional fields which can be used in Authn MFE. The fields that are in `EXTRA_FIELDS <https://github.com/openedx/edx-platform/blob/a9355852edede9662762847e0d168663083fc816/openedx/core/djangoapps/user_authn/api/helper.py#L20-L39>`_ are already exists as column in user profile model, but are disbaled, so when adding other field that don't exits, a step to do data migration is required.
+
+In order to add fields that already exist in the user model, it is enough to redefine several constants. How to do this will be described in instructions **A**. If you need to add fields that are not in the user model by default, then you need to use instruction **B**.
+
+**A. Add fields that are already exists as column in user profile model**
+-----------------
+You need to redefine several constants in the settings. Several methods can be used for this:
+
+#. **Redefine constants using Django admin and `Site configurations` API. (recommended):**
+    - Go to `Site configurations` in admin: {{LMS}}/admin/site_configuration/siteconfiguration/
+    - Add new settings to OrderedDict (create new `Site configurations` if it wasn't before):
+    .. code-block:: json
+    {
+        "ENABLE_DYNAMIC_REGISTRATION_FIELDS": "true",
+        "MFE_CONFIG": {
+            "ENABLE_DYNAMIC_REGISTRATION_FIELDS": "true"
+        },
+        "REGISTRATION_EXTRA_FIELDS": {
+            "country": "required",
+            "gender": "required"
+        }
+    }
+    
+    - All possible fields can be find in `EXTRA_FIELDS <https://github.com/openedx/edx-platform/blob/a9355852edede9662762847e0d168663083fc816/openedx/core/djangoapps/user_authn/api/helper.py#L20-L39>`_.
+    - REST API gets cached, if you are in hurry though you can do this command: `tutor local exec redis redis-cli flushall`.
+    - Or you can wait a few minutes until the cache is invalidated.
+    - After this, the new fields should appear on the Auth page.
```

Comment:
NIT:
Can we also mention that the required fields will appear on the registration form and optional fields will appear on progressive profiling form?